python/requirements.txt

```diff
@@ -1,7 +1,7 @@
 async-timeout==4.0.2
 google-api-python-client==2.85.0
 maturin==0.13.0
-protobuf==3.20.*
+protobuf==4.25.*
```

Comment:
```suggestion
protobuf==3.25.*
```

Reply:
such version does not exist https://pypi.org/project/protobuf/#history